source/_integrations/homeassistant_analytics.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: Home Assistant analytics
+description: Instructions on how to integrate Home Assistant analytics within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: '2024.2'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@joostlek'
+ha_domain: homeassistant_analyitcs
+ha_platforms:
+  - sensor
+ha_integration_type: service
+---
+
+The **Home Assistant analytics** {% term integration %} allows you to get integration usage statistics into Home Assistant.
```

Comment:
Besides the name changes, please add to the docs that only integrations with a reported usage bigger than 0 will be displayed. Meaning it can take some time until a new integration will be showing in the analytics

Credits @ludeeus